content/developer/automatedtesting/_index.en.adoc

```diff
@@ -10,149 +10,179 @@ title: "Automated Testing"
 
 == Introduction
 
-Automated testing provides a means to ensure that the high quality of SuiteCRM is maintained. SuiteCRM has an automated test suite which is powered by the http://codeception.com[codeception] testing framework.
-
-=== Installing the testing framework
-
-In your terminal emulator or command prompt:
-
-`cd path/to/suitecrm/instance`
-
-Install dependencies from Composer:
-
-`composer install`
+Automated testing provides a means to ensure that the high quality of SuiteCRM is maintained. SuiteCRM has an automated test suite which is powered by the http://codeception.com[Codeception] and https://phpunit.de[PHPUnit] testing frameworks.
 
 == Test Suites
 
-SuiteCRM offers many different test suites.
+SuiteCRM offers four different test suites.
 
 [width="80",cols="30,50",options="header",]
 |=======================================================================
 | Suite      | Description
 
-| install    | Acceptance test to test that the install wizard is working correctly
+| install    | Acceptance test verify that the install wizard is working correctly
 | acceptance | Automated browser testing, also known as 'feature tests'
-| unit       | Unit tests reflect that a single unit of code, e.g. a method, is working
+| unit       | Unit tests verify that a single unit of code, e.g. a method, is working correctly
 | api        | Functional tests which test the API version 8 responses
 |=======================================================================
 
+== Setup
 
-=== Running Codeception for the first time
-Codeception and the other commands live inside the `vendor/bin/` directory of your SuiteCRM instance. To test that your configuration is working you need to first install your copy of SuiteCRM. After installing SuiteCRM and downloading ChromeDriver you can run the acceptance tests:
+This guide assumes you have already installed and set up a supported PHP version, MySQL 5.7, and https://getcomposer.org[Composer].
 
-`./vendor/bin/codecept run acceptance`
+You can set up the CRM for testing with these steps:
 
+- In your terminal emulator or command prompt: `cd path/to/suitecrm/instance`
+- Install dependencies with https://getcomposer.org[Composer]: `composer install`
+- Create a MySQL database called `automated_tests`, along with a user called `automated_tests` and the password `automated_tests`. In the MySQL command line interface:
+  - `mysql -e "CREATE DATABASE automated_tests CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci;"`
+  - `mysql -u root -p -e "CREATE USER 'automated_tests'@'localhost' IDENTIFIED BY 'automated_tests';"`
+  - `mysql -u root -p -e "GRANT ALL PRIVILEGES ON automated_tests.* TO 'automated_tests'@'localhost';"`
+- Configure environment variables, see environment variables section below for more information.
```

Comment:
Dump question, how do you link to another section in the same page with asciidoc?